console/react/test_data/qdrService.mock.js

```diff
@@ -60,17 +62,13 @@ export const mockService = ({ onSendMethod }) => {
             resolve();
           });
         },
-        startUpdating: () => {
-          return new Promise(resolve => {
-            resolve();
-          });
-        },
```

Comment:
Fix for `63:9  warning  Duplicate key 'startUpdating'  no-dupe-keys`

Reply:
It's probably better to remove the startUpdating on line 52 and leave this one here. By returning a promise and then calling resolve(), this will allow the code that calls startUpdating().then(...) to run the code contained in the then(...).

Reply:
That's what I've done, except I've organized it that startUpdating and stopUpdating are next to each other. The new line 52 reads `startUpdating: () => new Promise(resolve => resolve()),` which is exactly what was here, except written as oneliner.

Reply:
... uh, no, it isn't exactly the same, the difference is between `return resolve()` and just `resolve()`. But it may not matter...

Reply:
return resolve() is odd, but it should still work.